### PR TITLE
Update meta-omics special interest group to include diagram of associated pipelines

### PR DESCRIPTION
**Pull request by jfy133**

### CODE REVIEW

File: sites/main-site/src/content/special-interest-groups/meta-omics/index.md [hunk 1]
```diff
@@ -30,7 +30,9 @@ The initial aims of this interest group will be:
 1. Helping coordinate compatibility between existing and potential meta-omic nf-core pipelines
 2. Implementing semi-'chaining' of sequential pipelines
 3. Developing standard parameters for different sub-disciplines (i.e. microbiome analysis) for each pipeline
-4. Documenting usage of 'first level' meta-omic pipelines followed by complementary analyses with "secondary" pipelines such as #funcscan, #phageannotator, #phyloplace, #rnaseq, #differentialabundance etc.
+4. Documenting usage of 'first level' meta-omic pipelines (createtaxdb, ampliseq, eager, taxprofiler, mag, metatdenovo) followed by complementary analyses with "secondary" pipelines such as (funcscan, phageannotator, phyloplace, rnaseq, differentialabundance) etc.
```
Comment (JoseEspinosa) [inline sites/main-site/src/content/special-interest-groups/meta-omics/index.md:36]:
```suggestion
4. Documenting usage of 'first level' meta-omic pipelines (createtaxdb, ampliseq, eager, taxprofiler, mag, metatdenovo) followed by complementary analyses with "secondary" pipelines such as funcscan, phageannotator, phyloplace, rnaseq, differentialabundance, etc.
```

File: sites/main-site/src/content/special-interest-groups/meta-omics/index.md [hunk 2]
```diff
@@ -30,7 +30,9 @@ The initial aims of this interest group will be:
...
+
+![London Underground style 'tube map' showing how potentially nf-core pipelines can be used for complementary and sequential analysis within meta-omics disciplines](nf-core-meta-omics-metro.png)
```
Comment (JoseEspinosa) [inline sites/main-site/src/content/special-interest-groups/meta-omics/index.md]: I think this version might stress better that the important bit is what is showing and not the metro map, which might be 😁 
```suggestion
![Showcasing how nf-core pipelines can potentially be used for complementary and sequential analysis within meta-omics disciplines using a London Underground style 'tube map'](nf-core-meta-omics-metro.png)
```

**Review by JoseEspinosa — APPROVED** [review]
Two small suggestions

File: sites/main-site/src/content/special-interest-groups/meta-omics/index.mdx [hunk 3]
```diff
@@ -30,7 +33,17 @@ The initial aims of this interest group will be:
...
 3. Developing standard parameters for different sub-disciplines (i.e. microbiome analysis) for each pipeline
-4. Documenting usage of 'first level' meta-omic pipelines followed by complementary analyses with "secondary" pipelines such as #funcscan, #phageannotator, #phyloplace, #rnaseq, #differentialabundance etc.
+4. Documenting usage of 'first level' meta-omic pipelines (createtaxdb, ampliseq, eager, taxprofiler, mag, metatdenovo) followed by complementary analyses with "secondary" pipelines such as (funcscan, phageannotator, phyloplace, rnaseq, differentialabundance) etc.
+
+<Image
+    src={metro-omics-metro}
+    class="d-block m-auto"
+    width={400}
+    density={[1.5, 2]}
+    alt="London Underground style 'tube map' showing how potentially nf-core pipelines can be used for complementary and sequential analysis within meta-omics disciplines"
```
Comment (jfy133) [inline sites/main-site/src/content/special-interest-groups/meta-omics/index.mdx:43]:
```suggestion
    alt="Diagram showcasing how nf-core pipelines can potentially be used for complementary and sequential analysis within meta-omics disciplines using a London Underground style 'tube map'"
```

File: sites/main-site/src/content/special-interest-groups/meta-omics/index.mdx [hunk 4]
```diff
@@ -30,7 +33,17 @@ The initial aims of this interest group will be:
 1. Helping coordinate compatibility between existing and potential meta-omic nf-core pipelines
 2. Implementing semi-'chaining' of sequential pipelines
 3. Developing standard parameters for different sub-disciplines (i.e. microbiome analysis) for each pipeline
-4. Documenting usage of 'first level' meta-omic pipelines followed by complementary analyses with "secondary" pipelines such as #funcscan, #phageannotator, #phyloplace, #rnaseq, #differentialabundance etc.
+4. Documenting usage of 'first level' meta-omic pipelines (createtaxdb, ampliseq, eager, taxprofiler, mag, metatdenovo) followed by complementary analyses with "secondary" pipelines such as (funcscan, phageannotator, phyloplace, rnaseq, differentialabundance) etc.
```
Comment (jfy133) [inline sites/main-site/src/content/special-interest-groups/meta-omics/index.mdx:36]:
```suggestion
4. Documenting usage of 'first level' meta-omic pipelines (createtaxdb, ampliseq, eager, taxprofiler, mag, metatdenovo) followed by complementary analyses with "secondary" pipelines (e.g., funcscan, phageannotator, phyloplace, rnaseq, differentialabundance) etc.
```

File: sites/main-site/src/content/special-interest-groups/meta-omics/index.mdx [hunk 5]
```diff
@@ -30,7 +33,17 @@ The initial aims of this interest group will be:
...
+
+<Image
+    src={metro-omics-metro}
+    class="d-block m-auto"
+    width={400}
+    density={[1.5, 2]}
+    alt="Diagram showcasing how nf-core pipelines can potentially be used for complementary and sequential analysis within meta-omics disciplines using a London Underground style 'tube map'"
+/>
+
+![](nf-core-meta-omics-metro.png)
```
Comment (jfy133) [inline sites/main-site/src/content/special-interest-groups/meta-omics/index.mdx:46]:
```suggestion
```

File: sites/main-site/src/content/special-interest-groups/meta-omics/index.md (hunk 2 above)
Comment (jfy133) [reply sites/main-site/src/content/special-interest-groups/meta-omics/index.md]: Updated!

File: sites/main-site/src/content/special-interest-groups/meta-omics/index.md (hunk 1 above)
Comment (jfy133) [reply sites/main-site/src/content/special-interest-groups/meta-omics/index.md]: Updated!

**Pull request merged**